run-e2e.sh

```diff
@@ -29,6 +29,7 @@ case "$1" in
     ;;
   cluster-loader2 )
     #CLUSTERLOADER2
+    echo "${PERFTEST_ROOT}/clusterloader2 && ./run-e2e.sh ${@:2}"
```

Comment:
Maybe you should add an echo above the switch case (in case we aren't even entering here).

Reply:
Done.